README.md

```diff
@@ -9,11 +9,11 @@
 [![Latest Stable Version](https://poser.pugx.org/yiisoft/middleware-dispatcher/v/stable.png)](https://packagist.org/packages/yiisoft/middleware-dispatcher)
 [![Total Downloads](https://poser.pugx.org/yiisoft/middleware-dispatcher/downloads.png)](https://packagist.org/packages/yiisoft/middleware-dispatcher)
 [![Build status](https://github.com/yiisoft/middleware-dispatcher/workflows/build/badge.svg)](https://github.com/yiisoft/middleware-dispatcher/actions?query=workflow%3Abuild)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/yiisoft/middleware-dispatcher/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/middleware-dispatcher/?branch=master)
```

Comment:
We remove scrutinizer config (`.scrutinizer.yml`), but keep scrutinizer analyze and badge "code quality".